src/vconsole/vconsole-setup.c

```diff
@@ -572,6 +582,13 @@ static int verify_source_vc(char **ret_path, const char *src_vc) {
         if (r < 0)
                 return log_error_errno(r, "Virtual console %s is not in K_XLATE or K_UNICODE: %m", src_vc);
 
+        /* setfont(8) silently ignores when the font can't be applied due to the vc being in
+         * KD_GRAPHICS. Hence we continue to accept this case however we now let the user know that the vc
+         * will be initialized only partially.*/
+        r = verify_vc_kdmode(fd);
+        if (r < 0)
+                log_notice("Virtual console %s is not in KD_TEXT, font settings won't be applied.", src_vc);
```

Comment:
as above

Reply:
and i'd tone down the message a bit. What you describe here is how setfont() operates today, maybe this is changed one day. i.e. just say "font settings likely won't be applied".